conf/application.conf

```diff
@@ -68,7 +68,7 @@ assets {
 retryCount = 3
 retryDelayMilliseconds = 500
 
-hasSandbox = false
+hasSandbox = true
```

Comment:
Should this be in here?

Reply:
I think this should be here!
You basically can't do any local dev with a sandbox apps unless that is set too true. So I think we should leave it set like this for local dev. It's overridden in the app-config-base to false (and I expect some in app-config-{env} it's set to true.